app.js

```diff
@@ -2,12 +2,14 @@ import express from "express";
 
 import routes from "./router";
 import "./helper/db";
+import Socket from "./helper/socket";
 
 class App {
   constructor() {
     this.server = express();
     this.middlewares();
     this.routes();
+    this.socket = new Socket();
```

Comment:
Acho que não tem necessidade de criar essa variável, já que a classe `Socket` não tem nenhum método
```suggestion
     new Socket();
```